core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -932,7 +932,10 @@ public NativeImageInvokerInfo build() {
                 }
 
                 List<NativeConfig.MonitoringOption> monitoringOptions = new ArrayList<>();
-                monitoringOptions.add(NativeConfig.MonitoringOption.HEAPDUMP);
+                if (!SystemUtils.IS_OS_WINDOWS || containerBuild) {
```

Comment:
Could you use SmallRye Common OS instead? Something like `OS.WINDOWS.isCurrent()`.

I'm not asking to change it everywhere but let's use this one when we add new code.

Reply:
> Could you use SmallRye Common OS instead? Something like OS.WINDOWS.isCurrent().

Sure.

> I'm not asking to change it everywhere but let's use this one when we add new code.

I think I prefer changing it everywhere so that future me (and others I hope) will see what's being used and not mistakenly use `SystemUtils` :)